content/post/data-analyst.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Coolest Job in Tech"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["Analytics", "Org Structure"]
+date: 2020-01-05T21:16:09-05:00
+draft: true
+---
+
+Data analysts at early stage, tech-adjacent start ups have the coolest jobs. For junior and mid level folks especially, there’s so much to learn (hopefully an exciting proposition!)  as analysts  wear many different hats. In addition, there are numerous career paths to take once an analyst has cut their teeth for a few years. Due to its broad set of responsibilities, an analyst’s job can be esoteric and often means different things to different people.
+
+Folks outside of Data sometimes struggle with answering the question, “What does Data do here?” Folks can reason around “analysis”, “reporting” and “insights”, but otherwise their understanding is usually surface level. I’ve found it helpful to explain what the Analytics team does in terms of other job functions with which they might be more familiar.
```

Comment:
Folks twice scans oddly for me -- would change one to "People" or at least change the sentence structure so it's not the first word both times